modules/nodes/variables.tf

```diff
@@ -247,6 +250,7 @@ variable "default_node_groups" {
 variable "additional_node_groups" {
   description = "Additional EKS managed node groups definition."
   type = map(object({
+    node_group_type       = optional(string, null)
```

Comment:
Nodes beyond those explicitly defined as `platform` will undergo a rotation process with a 50% unavailability setting. I wasn't sure how we wanted to handle "additional nodes"